.github/workflows/update-website.yml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: "Update Website"
+on:
+  push:
+    # run on pushes to main (after merging release branches)
+    branches:
+      - main
```

Comment:
do we want a similar action for develop branch so we always have "bleeding edge" version of the code deployed somewhere for observation?

Reply:
I ended up adding it to this PR, which I made last week: https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/pull/785